code/parachain/frame/vesting/src/lib.rs

```diff
@@ -34,9 +34,9 @@
 		clippy::disallowed_types,
 		clippy::indexing_slicing,
 		clippy::todo,
-		clippy::unwrap_used,
-		clippy::panic
-	)
+		clippy::expect_used
+	),
+	deny(clippy::unwrap_used, clippy::panic)
```

Comment:
What's the expected effect here?

Reply:
To forbid `unwrap` and `panic` and warn `expect`s.

Reply:
In the build we don't allow "warn" level stuff also, @benluelo is this line needed?

Reply:
Well I'll better remove those warnings as things like `#[pallet::pallet]` and the `expect`s from the `GenesisBuild` will trigger them.

Reply:
`expect` should be allowed, and all uses of it should be audited